packages/@aws-cdk/core/lib/cfn-parse.ts

```diff
@@ -30,6 +30,24 @@ import { undefinedIfAllValuesAreEmpty } from './util';
  * @experimental
  */
 export class FromCloudFormation {
+  /**
+   * Creates a new object by removing all entries with the given keys from the argument object.
+   *
+   * @param object the object whose keys should be filtered out
+   * @param props the list of property names which should be removed from `object`
+   * @returns a new object with all the entries in `object`,
+   *   except those whose keys are one of the names provided in `props`
+   */
+  public static omit(object: object, ...props: string[]): object {
```

Comment:
The location of this function is a tough un' eh.

It strictly speaking has nothing to do with `FromCloudFormation`, so it should be a generic `Util` function. But on the other hand, it is only used there and it's nice and convenient to put it here. Oh well, I guess?

Reply:
Yeah, it was put here more for convenience than anything else.

The only other place I can think of that makes sense for it is in `runtime.ts` in `@aws-cdk/core`.

Let me know what you think!

Reply:
I think `runtime.ts` might make for a good location, yeah.